pkgs/by-name/fr/froide/package.nix

```
@@ -0,0 +1,174 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  makeWrapper,
+  gdal,
+  geos,
+  pnpm,
+  nodejs,
+  postgresql,
+  postgresqlTestHook,
+  playwright-driver,
+}:
+let
+
+  python = python3Packages.python.override {
+    packageOverrides = self: super: { django = super.django.override { withGdal = true; }; };
+  };
+
+in
+python.pkgs.buildPythonApplication rec {
+  pname = "froide";
+  version = "0-unstable-2024-11-22";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "okfde";
+    repo = "froide";
+    rev = "a90f5c4d40b46a161111eefdc84e5214e85715b0";
+    hash = "sha256-Q+iNI3yqxqAtDONHY+SaZeMyjY6hqTxwy7YmiiY94+0=";
+  };
+
+  patches = [ ./django_42_storages.patch ];
+
+  pythonRelaxDeps = [
+    "pikepdf"
+    "channels"
+  ];
+
+  build-system = [ python.pkgs.setuptools ];
+
+  nativeBuildInputs = [
+    makeWrapper
+    nodejs
+    pnpm.configHook
+  ];
+
+  dependencies = with python.pkgs; [
+    bleach
+    celery
+    celery-singleton
+    channels
+    coreapi
+    dj-database-url
+    django
+    django-celery-beat
+    django-celery-email
+    django-configurations
+    django-contrib-comments
+    django-crossdomainmedia
+    django-elasticsearch-dsl
+    django-filingcabinet
+    django-filter
+    # Project discontinued upstream
+    # https://github.com/okfde/froide/issues/893
+    django-fsm
+    django-json-widget
+    django-leaflet
+    django-mfa3
+    django-oauth-toolkit
+    django-parler
+    django-storages
+    django-taggit
+    django-treebeard
+    djangorestframework
+    djangorestframework-csv
+    djangorestframework-jsonp
+    drf-spectacular
+    drf-spectacular-sidecar
+    easy-thumbnails
+    elasticsearch
+    elasticsearch-dsl
+    geoip2
+    icalendar
+    markdown
+    phonenumbers
+    pillow
+    pikepdf
+    psycopg
+    pygtail
+    pyisemail
+    pypdf
+    python-magic
+    python-mimeparse
+    python-slugify
+    requests
+    wand
+    weasyprint
+    websockets
+  ];
+
+  pnpmDeps = pnpm.fetchDeps {
+    inherit pname version src;
+    hash = "sha256-DMoaXNm5S64XBERHFnFM6IKBkzXRGDEYWSTruccK9Hc=";
+  };
+
+  postBuild = ''
+    pnpm run build
+  '';
+
+  postInstall = ''
+    cp -r build manage.py $out/${python.sitePackages}/froide/
+    makeWrapper $out/${python.sitePackages}/froide/manage.py $out/bin/froide \
+      --prefix PYTHONPATH : "$PYTHONPATH" \
+      --set GDAL_LIBRARY_PATH "${gdal}/lib/libgdal.so" \
```

Should this use
```suggestion
      --set GDAL_LIBRARY_PATH "${gdalMinimal}/lib/libgdal.so" \
```
just like postgis?